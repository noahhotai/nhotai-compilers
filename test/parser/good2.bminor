main: function void () = {
	print "hi";
}