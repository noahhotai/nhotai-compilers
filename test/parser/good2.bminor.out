IDENTIFIER     main
COLON :
FUNCTION            function
VOID           void
LEFT PARENTHEIS     (
RIGHT PARENTHESIS    )
ASSIGN         =
LEFT BRACE     {
PRINT          print
STRING LITERAL hi
SEMICOLON      ;
RIGHT BRACE    }
Program parsed successfully.
